api/Controllers/UsersController.cs

```diff
@@ -27,4 +33,48 @@ public override Task<IActionResult> GetAll()
     {
         return base.GetAll();
     }
+
+    /// <summary>
+    /// Get user by email
+    /// </summary>
+    /// <returns>User matching email</returns>
+    [HttpGet]
+    [Route("email")]
+    public async Task<IActionResult> GetByEmail(string email)
+    {
+        var matchingUser = await base.Service.GetWithPermissionsAsync(email);
+        if (matchingUser != null)
+        {
+            return Ok(matchingUser);
+        }
+        return NoContent();
+    }
+
+    /// <summary>
+    /// Allows a new user without authorization to JASPER to request access to the application.
+    /// </summary>
+    /// <returns>The user resulting from the access request.</returns>
+    [Authorize(AuthenticationSchemes = "SiteMinder, OpenIdConnect", Policy = nameof(ProviderAuthorizationHandler))]
+    [HttpGet]
+    [Route("request-access")]
+    public async Task<ActionResult> RequestAccess(string email)
+    {
+        var userIdentifier = User.Claims.FirstOrDefault(c => c.Type == "preferred_username")?.Value;
+        logger.LogInformation("User Id {UserId}, requested access with guid: {UserIdentifier}", User.UserId(), userIdentifier);
+
+        Guid userGuid;
+        Guid.TryParse(userIdentifier?.Split("@").FirstOrDefault(), out userGuid);
+        var newUser = new UserDto()
+        {
+            FirstName = User.Claims.FirstOrDefault(c => c.Type == "given_name")?.Value,
```

Comment:
Might better to use the code below to avoid hard coded strings.
                        
```suggestion
           FirstName = User.FindFirstValue(ClaimTypes.GivenName),
           LastName = User.FindFirstValue(ClaimTypes.Surname),
```